samples/spring-security-saml-dsl-sample/src/main/resources/application.yml

```diff
@@ -2,6 +2,10 @@ server:
   port: 8443
   ssl:
     enabled: true
-    key-alias: colombia
-    key-store: src/main/resources/saml/colombia.jks
-    key-store-password: colombia-password
\ No newline at end of file
+    key-alias: spring
+    key-store: src/main/resources/saml/keystore.jks
+    key-store-password: secret
+
+security:
+  saml2:
+      metadata-url: https://dev-348145.oktapreview.com/app/exk6bmtlqpOBRo4a20h7/sso/saml/metadata
```

Comment:
There are no tests against this right now. However, we can add some. Likely we should create a specific okta account for running the tests. We would keep the admin account private, but add an account for authenticating in the test.